src/SankeyDiagram.js

```diff
@@ -849,13 +899,65 @@ export default class SankeyDiagram extends React.Component {
       );
     }
 
+    function displayStepLabelsIf(
+      stepLabelText,
+      stepLabelClassName,
+      stepLabelStyle,
+      stepLabelPadding,
+      nodes
+    ) {
+      if (!stepLabelText) {
+        return null;
+      }
+
+      const depthMapXPos = {};
+      const depthMapYPos = {};
+
+      nodes.forEach(n => {
```

Comment:
are nodes dom elements?

Reply:
these are the nodes returned to us after creating the Sankey graph using d3

https://github.com/d3/d3-sankey#sankey_nodes

Reply:
I just wanted to make sure that this isn't a DOM element array, as they don't have the same prototypical functions as regular arrays